brainstormforce/astra
# AST-1618 - Global typography section takes time to expand

### Description
- Due to number of controls inside, the Global > Typography control take time
- Revamped open typo controls into sub-controls of settings-group to make it lazy load
- Due to that implementation of settings-group, typo controls will lazily load 
- Migrated variant control from PHP to React.

### Screenshots
- https://share.bsf.io/4guKW8YQ
- Speed - https://share.bsf.io/bLuKbgze

### Types of changes
- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
- Check master version - Global > Typography section (12 sec, local dev setup)
- In current version - Global > Typography section (1 sec, same local setup)
- Check variant control backward, new setup, different fonts etc

### Checklist:
